src/components/ATATTopNavBar.vue

```diff
@@ -149,7 +150,10 @@ export default class ATATTopNavBar extends Vue {
         // open external URL in new tab
         window.open(item.externalUrl, "_blank");
       }
-    }
+
+      //closes slideoutpanel if necesssary
+      SlideoutPanel.closeSlideoutPanel();
```

Comment:
I believe this is already being handled after my latest branch - on navigation, slideout drawer is closed automatically now

Reply:
removed